prov/efa/src/efa_strerror.c

```diff
@@ -51,11 +51,19 @@ static const char *efa_io_comp_status_str(enum efa_errno status)
 		[FI_EFA_LOCAL_ERROR_BAD_LENGTH]        = "Message too long",
 		[FI_EFA_REMOTE_ERROR_BAD_ADDRESS]      = "Invalid address",
 		[FI_EFA_REMOTE_ERROR_ABORT]            = "Receiver connection aborted",
-		[FI_EFA_REMOTE_ERROR_BAD_DEST_QPN]     = "Invalid receiver queue pair number (QPN)",
+		[FI_EFA_REMOTE_ERROR_BAD_DEST_QPN]     = "Invalid receiver queue pair number (QPN). "
+		                                         "This error can be caused by a crashed peer. "
+		                                         "Please verify the peer application has not "
+		                                         "terminated unexpectedly.",
 		[FI_EFA_REMOTE_ERROR_RNR]              = "Receiver not ready",
 		[FI_EFA_REMOTE_ERROR_BAD_LENGTH]       = "Receiver scatter-gather list (SGL) too short",
 		[FI_EFA_REMOTE_ERROR_BAD_STATUS]       = "Unexpected status received from remote",
-		[FI_EFA_LOCAL_ERROR_UNRESP_REMOTE]     = "Unresponsive receiver",
+		[FI_EFA_LOCAL_ERROR_UNRESP_REMOTE]     = "Unresponsive receiver. "
+		                                         "This error can be caused by a peer hardware failure or "
```

Comment:
"can be" -> "is typically caused either peer hardware failure or incorrect inbound/outbound rule"